pkg/controller/telemetry/metric.go

```diff
@@ -203,9 +203,15 @@ func buildMetricFromWorkload(dstWorkload, srcWorkload *workloadapi.Workload) com
 	}
 
 	trafficLabels := commonTrafficLabels{}
+	services := dstWorkload.Services
+	svcHost := ""
+	for k := range services {
+		svcHost = strings.Split(k, "/")[1]
+	}
+	svcName := strings.Split(svcHost, ".")[0]
```

Comment:
ditto

Reply:
This is not correct for serviceEntry, the hostname defined in SE may have nothing to do with *.cluster.local.

Reply:
Do you mean the hostname might be www.XXX.com?

Reply:
yeah

Reply:
I donot mind we use FQDN if service name is a little bit hard to acquire